packages/jest-runner/src/index.js

```diff
@@ -89,17 +95,12 @@ class TestRunner {
     onResult: OnTestSuccess,
     onFailure: OnTestFailure,
   ) {
-    const farm = workerFarm(
-      {
-        autoStart: true,
-        maxConcurrentCallsPerWorker: 1,
-        maxConcurrentWorkers: this._globalConfig.maxWorkers,
-        maxRetries: 2, // Allow for a couple of transient errors.
-      },
-      TEST_WORKER_PATH,
-    );
+    // $FlowFixMe: "default" property will be exposed.
```

Comment:
It would be cool if `default` (as long as `__esModule: true`) would be the actual default export instead of attached to the `default` key (so you could just do `worker(stuff)`).

Reply:
I have been thinking about this, and I'm happy to see feedback on that direction. The reason of the `default` comes from the fact that a class instance is returned from `new`. The change here would be to make `constructor` return a method, but then `instanceof JestWorker` wouldn't work.

I will think more about it and see what's the best way of fixing the issue; but generally speaking, I agree with your comment.